Jenkinsfile

```diff
@@ -0,0 +1,105 @@
+    }
+    options {
+        timeout(time: 30, unit: 'MINUTES')
+    }
+    environment {
+        jenkinsOsxPub = credentials('jenkins.osx.pub')
+        jenkinsOsxPriv = credentials('jenkins.osx.priv')
+        artifactoryUsername = credentials('artifactoryUsername')
+        artifactoryPassword = credentials('artifactoryPassword')
+        AWS_ACCESS_KEY_ID = credentials('AWS_ACCESS_KEY_ID')
+        AWS_SECRET_ACCESS_KEY = credentials('AWS_SECRET_ACCESS_KEY')
+        STARDOG_GH_CREDS = credentials('stardog-ops-github-jenkins')
+        STARDOG_LICENSE = credentials('stardog_license_base64')
+        TF_VAR_artifactory_url = 'http://stardog.jfrog.io/stardog/stardog-releases-local'
+        TF_VAR_snapshot_url = 'http://stardog.jfrog.io/stardog/stardog-snapshots-local'
+        TF_VAR_artifactory_username = credentials('artifactoryUsername')
+        TF_VAR_artifactory_password = credentials('artifactoryPassword')
+        STARDOG_GITHUB_API_TOKEN = credentials('stardog-github-api-token')
+
+        artifactoryUsernameRO = credentials('artifactory-ci-readonly')
+
+        LANG = 'C.UTF-8'
+        S3_LOCAL_HOSTNAME = "s3${env.BUILD_ID}${UUID.randomUUID().toString().replace("-", "")}"
+        RHEL_CREDS = credentials('RHEL')
+    }
+    stages {
+        stage('Setup env') {
+            steps {
+                script {
+                    env.ARTIFACT_TAG = params.ARTIFACT_TAG.toLowerCase()
+                    setJavaVersion()
+                }
+            }
+        }
+        stage('Build') {
+            steps {
+                script {
+                    // The default Java version in the image is 11, which should be used to build
+                    printJavaVersion()
+                    sh "sbt assembly"
+                }
+            }
+        }
+        stage('Publish jars and zip internally') {
+            steps {
+                script {
+                    sh "sudo chmod -R 755 ./ops"
+                    sh "./ops/jenkins/delete-nightly-snapshot.sh"
+                    sh "./ops/jenkins/publish_artifacts.sh"
+                }
+            }
+        }
+        stage('Slack Message') {
+            when { expression { params.SLACK != "" } }
+            steps {
+                script {
+                    notifyBuild("Success :discodancer:", "GREEN", params)
+                }
+            }
+        }
+    }
+    post {
+        failure {
+            script {
+                if (params.SLACK != "") {
+                    notifyBuild("Failed :computerrage:", "RED", params)
+                }
+            }
+        }
+    }
+}
+
+def setJavaVersion() {
+    echo "Setting Java version to Java 11"
+    env.JAVA_HOME = "/usr/lib/jvm/java-11-openjdk-amd64/"
+    sh "sudo update-java-alternatives --set java-1.11.0-openjdk-amd64"
+    printJavaVersion()
+}
+
+def printJavaVersion() {
+    echo "Checking Java version"
+    echo "JAVA_HOME=${env.JAVA_HOME}"
+    sh "java -version"
+}
```

Comment:
I don't think we need all this java stuff, unless you need to add support for 8. But otherwise we can just use the default of 11.

Reply:
Yes, I had it for debugging the installation. Now removed